src/formatters/SimpleCellFormatter.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 export interface SimpleCellFormatterProps {
-  value: string | number | boolean;
+  value?: string | number | boolean;
 }
 
 export function SimpleCellFormatter({ value }: SimpleCellFormatterProps) {
```

Comment:
Should change div -> span.
Blocks prevent text ellipsis from working.

![image](https://user-images.githubusercontent.com/567105/71735578-beafc480-2e46-11ea-9550-7ca8e3e7e30c.png)
![image](https://user-images.githubusercontent.com/567105/71735589-c4a5a580-2e46-11ea-916a-2728026c7385.png)

Reply:
Done